notebooks/vertex_matching_engine/solutions/matching_engine_ann.ipynb

```diff
@@ -0,0 +1,646 @@
+{
```

Comment:
<p>Is it possible to explain why using a VPC reduces latency in a brief sentence?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/230/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>